packages/components/pages/f-offers/src/components/NoOffersFound.vue

```diff
@@ -88,16 +93,45 @@ export default {
     & .c-noOffersFound-searchBox {
         padding-top: spacing(d);
 
+        & input {
+            background-color: $color-white;
+            height: 61px;
+        }
+
         @include media('<=mid') {
             width: 300px;
         }
 
         @include media('<=narrowMid') {
             width: 100%;
+            & input {
+                height: 58px;
+            }
         }
 
-        & input {
-            background-color: $color-white;
+        @include media('<=narrow') {
+            width: 100%;
+            & input {
+                height: 48px;
+            }
+        }
+    }
+
+    .c-noOffersFound-searchBox--wrapper {
+        position: relative;
+    }
+
+    .c-noOffersFound-searchBox--background {
```

Comment:
This looks like it's applied to a child element rather than a modifier of the base element so I'd replace the double dashes with a single one.

Reply:
applied to wrapper also